tests/test_pycode/test_pycode_ast.py

```diff
@@ -51,6 +51,8 @@
      "lambda x=0, /, y=1, *args, z, **kwargs: ..."),  # posonlyargs
     ("0x1234", "0x1234"),                       # Constant
     ("1_000_000", "1_000_000"),                 # Constant
+    ("Tuple[:,:]", "Tuple[::, ::]"),            # Index, Subscript, Slice
```

Comment:
`[::,::]` feel weird and incorrect to me. For the entirely empty slice, I think `:` should be used instead (i.e., when node.lower, node.upper and node.step are not set).

Reply:
You're right - fixing that in a bit.